apps/fz_http/lib/fz_http_web/live/user_live/vpn_status_component.ex

```diff
@@ -0,0 +1,24 @@
+defmodule FzHttpWeb.UserLive.VPNStatusComponent do
+  @moduledoc """
+  Handles VPN status tag.
+  """
+  use FzHttpWeb, :live_component
+
+  @impl Phoenix.LiveComponent
+  def render(assigns) do
+    ~H"""
+    <label>
+    <%= cond do %>
+      <% @user.disabled_at -> %>
+      <span class="tag is-danger is-medium" title="This user's VPN connect is disabled by an administrator or OIDC refresh error">DISABLED</span>
```

Comment:
Disabled could also be caused by OIDC refresh failure, if the settings is turned on.

Reply:
https://github.com/firezone/firezone/pull/727/files#diff-919bbc71f8f553a102da75b2bbe47e4f71eb4be840a4c79bb4423c0ada4a662aR41

Reply:
This field on user should be set if the refresh fails, right? I think that's all I'd need to show the disabled tag.